css-text-3/line-breaking/reference/line-breaking-ic-001-ref.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title>Reference File for line breaking tests using ideographic caracters</title>
+<link rel=author title="Florian Rivoal" href="http://florian.rivoal.net">
+<!--
+	This is reference is theoretically identical to line-breaking-001.html,
+	but is needed nonetheless due to difference in anti-aliasing / font rendering.
```

Comment:
Also, while we're at it, you made me notice that I wrote "identical to line-breaking-001.html" instead of "identical to line-breaking-001-ref.html", so I added a commit to fix just that.